packages/core/src/app/customer/Customer.tsx

```diff
@@ -370,6 +370,8 @@ class Customer extends Component<CustomerProps & WithCheckoutCustomerProps & Ana
 
         try {
             await sendLoginEmail(values);
+        } catch (error) {
+        // Need to write catch block since one test covers the case when `sendLoginEmail` fails
```

Comment:
Do you have a ticket to handle the error?

Reply:
https://bigcommercecloud.atlassian.net/browse/CHECKOUT-8703